todo_app/tests/test_index_model.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+from todo_app.models.index_model import IndexModel
+from todo_app.classes.item import Item
+
+doing_item = Item('1', 'Title', 'Desc', status="Doing")
+to_do_item = Item('2', 'Title', 'Desc', status="To Do")
+done_item = Item('3', 'Title', 'Desc', status="Done")
+
+fake_items = [doing_item, to_do_item, done_item]
+
+
+@pytest.fixture
+def index_model(monkeypatch):
```

Comment:
not sure if this requires monkeypatch? tests do run without it

Reply:
Yep must've accidentally added that